src/main/java/Game.java

```diff
@@ -0,0 +1,54 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Game {
+
+    List<Car> carList = new ArrayList<Car>();
+    List<Car> winnerList = new ArrayList<Car>();
+
+    public void playGame(int carNum, int roundNum) {
+        initGame(carNum);
+        for (int i = 0; i < roundNum; i++) {
+            playRound();
+        }
+        makeWinnerList();
+    }
+
+    public void initGame(int carNum){
+        for (int i = 0 ; i < carNum ; i++)
+            carList.add(new Car("Car_"+i+""));
+    }
+
+    public void playRound() {
+        for (Car car : carList) {
+            car.move(getRandom());
+        }
+    }
+
+    public int getRandom(){
+        return (int)(Math.random()*10);
+    }
+
+    public void makeWinnerList() {
+        Car winner = carList.get(0);
+        int maxDistance = getMaxDistance();
+        for (Car car : carList) {
+            addWinner(car, maxDistance);
+        }
+    }
+
+    private void addWinner(Car car, int maxDistance) {
+        if (car.getDistance() == maxDistance) {
+            winnerList.add(car);
+        }
+    }
+
+    private int getMaxDistance() {
+
+        int maxDistance  = 0;
+        for (int i = 0; i < carList.size(); i++) {
+            maxDistance = Math.max(maxDistance, carList.get(i).getDistance());
+        }
```

Comment:
인덱스가 필요 없다면, 아래와 같이 enhanced for-loop 를 사용하는게 어떨까요!
```suggestion
        for (Car car : carList) {
            maxDistance = Math.max(maxDistance, car.getDistance());
        }
```